vite.config.ts

```diff
@@ -4,4 +4,7 @@ import react from '@vitejs/plugin-react'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
+    host:'0.0.0.0',
+  },
```

Comment:
where'd you get this, was this the magic trick?

Reply:
I think that by running it with docker-compose up and just watching the logs. 
It said sth about listening in 127.0.0.1